### PR TITLE
[Docs]: Add section describing the use of children and grandchildren 

### LINKED ISSUE #9165: [Docs] TOC: Add section describing the use of children and grandchildren
We need to document the use of the `grandchildren`<https://github.com/meshery/meshery/blob/03345f16a853f613b83b7d6e18c55552dd30b13e/docs/_data/toc.yml#L30-L31| stanza >that I added to the table of contents sidebar in the <https://docs.meshery.io/project/contributing/contributing-docs|Contributing to Docs> page.

[Slack Message](https://layer5io.slack.com/archives/C04LJS10VTL/p1697904313067449?thread_ts=1697904313.067449&cid=C04LJS10VTL)

### PR DESCRIPTION
**Pull request by coder12git**
**Notes for Reviewers**

This PR partially addresses #9165

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a64980b7ed64ef6e2c1fa1144183d48b7b4f000d at: https://65359151062e1b34ad119f8f--meshery-docs-preview.netlify.app

File: docs/assets/img/platforms/bash.png
Comment (leecalcote) [inline docs/assets/img/platforms/bash.png:1]: Oh! Very nice, @coder12git. Thank you for including these icons.

**Review by leecalcote — APPROVED** [review]
@coder12git, I that we discussed a slightly different enhancement to the contributing docs, that you will be chasing down, however, in the meantime, this is and ideal and enhancement, I think.

**Comment by leecalcote** [comment]
@coder12git, I updated the description of this pull request from "fixes #" to "partially addresses #9165", so that this original issue is not closed out, yet.

**Comment by leecalcote** [comment]
@aboobakersiddiqr63 heads-up...

**Comment by coder12git** [comment]
> @coder12git, I that we discussed a slightly different enhancement to the contributing docs, that you will be chasing down, however, in the meantime, this is and ideal and enhancement, I think.

@leecalcote , I apologize for any misunderstanding, here is the new PR - https://github.com/meshery/meshery/pull/9172

**Comment by coder12git** [comment]
> @coder12git please change the permalink of the pages from linux to linux-mac

Done @aboobakersiddiqr63

**Pull request merged**